client/src/App.js

```diff
@@ -1,27 +1,42 @@
 import React, {Component} from 'react';
 import './App.css';
-import Navbar from './components/Navbar';
 import { BrowserRouter as Router, Routes, Route} from 'react-router-dom';
 import { ReactSession } from 'react-client-session';
 
+//Imports pages to allow BrowserRouter to access them
 import Home from './pages';
 import Profile from './pages/profile';
 import SignUp from './pages/signup';
 import Login from './pages/login';
 import PasswordReset from './pages/passwordReset';
+import Logout from './pages/logout';
```

Comment:
Do not import logout page here.  Ryan and I already handled this on the view-profile PBI